README.md

```diff
@@ -147,6 +140,11 @@ Arduino's serial connection or
 [firmata](https://github.com/mobileoverlord/firmata) for communication using the
 Arduino's Firmata protocol.
 
+### How do I call Circuits I2C from Erlang?
+
+An Erlang friendly binding has been provided to simplify syntax when calling `Circuits.I2C` functions from Erlang code.  Instead of prefixing calls with: `'Elixir.Circuits.I2C':` you may use the binding: `circuits_i2c:`.
+For example: `circuits_i2c:open("i2c-1")`.  Example call to a function that can raise an exception: `circuits_i2c:'read!'(Ref, 16#40, 3)`.
```

Comment:
I can't find the Erlang equivalent for functions that raise exceptions. It doesn't seem idiomatic. `circuits_i2c:'read!'/3` looks really funny to me. If this were a pure Erlang library, do you know how these functions would be named? If that library wouldn't include them, then maybe we shouldn't wrap them.

Reply:
There is no idiomatic way to indicate Erlang functions that raise exceptions, that I know of.  This works, but your right, it looks funny.

Reply:
Do you think any Erlang programmers would actually call those functions? I'm leaning towards not wrapping those at all.

Reply:
If I had to say one way or the other, probably not.  I can take those out of the wrapper.  I don't have a strong preference either way.

Reply:
I'd like them out then. It will save me from having to explain why we included them to a couple Erlang programmers.

Reply:
Done.